tools/index.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
+    extractB3dm : require('./lib/extractB3dm'),
+    extractCmpt : require('./lib/extractCmpt'),
+    extractI3dm : require('./lib/extractI3dm'),
     glbToB3dm : require('./lib/glbToB3dm'),
     glbToI3dm : require('./lib/glbToI3dm'),
     gzipTileset : require('./lib/gzipTileset'),
-    pipeline : require('./lib/runPipeline')
+    optimizeGlb : require('./lib/gzipTileset'),
```

Comment:
I don't see the need to expose this one as it is equivalent to just including `gltf-pipeline` as a library.